### DISCUSSION TITLE
General

### DISCUSSION THREAD

**Discussion by ScriptedAlchemy** [module-federation/module-federation-examples#575] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jacob-ebey** [comment]
Welcome everyone. I'm Jacob and have been part of MF for a while now and am here to share ideas and push boundaries with you. Some would say "thought leader". I have lots of experience around using webpack / MF in node environments and currently have a focus around enabling distributed architectures on the server.

Useful utilities / resources:
- Stats plugin that supports federated modules to enable things like SSR preloading: https://github.com/jacob-ebey/webpack-federated-stats-plugin
- Adding startup code: https://gist.github.com/jacob-ebey/23aee3036c0c0c78a0b9369a5d8286ff
- Direct access to containers: https://gist.github.com/jacob-ebey/1a6edce80519c25b4725a05cc5618ee4
